src/main/java/co/rsk/bitcoinj/core/CheckpointManager.java

```diff
@@ -53,19 +53,19 @@
  *    headers from the genesis block.</li>
  * </ol>
  *
- * <p>Checkpoints are used by the SPV {@link BtcBlockChain} to initialize fresh
- * {@link org.bitcoinj.store.SPVBlockStore}s. They are not used by fully validating mode, which instead has a
+ * <p>Checkpoints are used by the SPV {@link BlockChain} to initialize fresh
```

Comment:
Is this change right?

Reply:
The reference `BlockChain` is wrong, that's for `bitcoin-core`.  Updated.